nebula_ros/launch/aeva_launch_all_hw.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="sensor_model" description="Aeries2"/>
```

Comment:
description or default?

Reply:
Thanks! Addressed in 3a7898a. Also added support for Aeva in the generic launch file in d7f1c7a.